Tidied up the code

sdushantha opened the pull request:
Hey there!

I want to contribute by making some minor changes to the code.
The list below covers the changes I made:
- fixed import order and removed unused imports
- removed white space and trailing space
- fixed spacing between equal signs

Overall, my intentions were to make the code comply more towards PEP8

sdushantha commented:
@superdiana Could you please let me know why this PR was closed? I'd be more than happy to fix it, if there is something wrong with the changes I have made.

superdiana commented:
@sdushantha Hello!, this was an honest mistake. I reopened the PR again and will review it one more time and provide feedback. Thanks for your contribution and happy hacking

superdiana commented:
I reopened this PR but this doesn't add value to the project, perhaps a linter GitHub action along with this would be helpful. If you want to contribute

sdushantha commented:
@superdiana I have added a linter GitHub action.
b32543b and 937bfcb should have been in the same commit, so I squashed them.

Let me know what you think about the changes I have made :)

superdiana commented on .github/workflows/linter.yml:
```diff
@@ -0,0 +1,30 @@
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```
What about the rest of the versions?

sdushantha replied:
> What about the rest of the versions?

From my knowledge, the Python version which `flake8` is installed on, has no affect on it's linting.
Running `flake8` with Python3.8 and Python3.6 should have no difference on the output given by `flake8`.

So I don't see the point of using different Python versions for the linting. 

I could be very wrong about this, so please do let me know if what I said is incorrect.

superdiana replied:
all good
